### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by nonemax**

### CODE REVIEW

**Comment by cap1984** [comment]
Are we going to use them inside our Jenkins test infrastructure ?

**Comment by nonemax** [comment]
> Are we going to use them inside our Jenkins test infrastructure ?

@delgod are we?

File: broker/controller_test.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+		},
+	}
+
+	for name, test := range tests {
+		t.Run(name, func(t *testing.T) {
+			err := c.updateInstanceMap(test.data)
+			if err != nil {
+				t.Error(name, err)
+			}
+			log.Println(len(c.instanceMap))
```
Comment (dAdAbird) [inline broker/controller_test.go]: ?

File: broker/controller_test.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+			if len(c.instanceMap) != test.instQuant {
+				t.Error(name+":", "Wrong instance quantoty")
```
Comment (dAdAbird) [inline broker/controller_test.go]: s/quantoty/quantity

**Review by dAdAbird — APPROVED** [review]
but there are a few minor issues - see comments

**Review by cap1984 — APPROVED** [review]

**Pull request merged**